### PR TITLE
WIP: Add a search bar in the dev pipeline.

### PR DESCRIPTION
**Pull request by jagtalon**
The search field goes through each name of the IA in the dev pipeline and hides `<li>` elements that don't match.

For @MariagraziaAlastra

### CODE REVIEW

File: src/templates/dev_pipeline.handlebars [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<input type="text" class="search-thing">
```
Comment (russellholt) [inline src/templates/dev_pipeline.handlebars]: I'll let @MariagraziaAlastra address this but you might want to call `search-thing` something more specific :palm_tree:

**Comment by MariagraziaAlastra** [comment]
Tested and it works, nice job @jagtalon! Now let's make this a little bit prettier, more like the search box in the index. Also I'd like to see the search input placed on the same line of the pager button on top, but on the opposite side, I think it would save space and look cleaner as well. What do you think?

**Comment by jagtalon** [comment]
@MariagraziaAlastra I think that would look good

**Comment by jagtalon** [comment]
@MariagraziaAlastra made the changes!

**Comment by MariagraziaAlastra** [comment]
Thanks @jagtalon! LGTM :+1:

**Comment by jagtalon** [comment]
Thanks!

**Pull request merged**